cmd/metal-api/internal/datastore/integer.go

```diff
@@ -15,15 +15,32 @@ var (
 	IntegerPoolRangeMax = uint(131072)
 )
 
+// IntegerPool manages unique integers
+type IntegerPool struct {
+	tablename string
+	min       uint
```

Comment:
As soon as `min` and `max` are introduced as private fields, the variables `IntegerPoolRangeMin` and `IntegerPoolRangeMax` are kind of obsolete or at least the name is weak (because they are not a real maximum).

I think, it would be better to name them `VRFPoolMin/Max` and `ASNPoolMin/Max`.

Reply:
Yes

Reply:
done